common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -38,8 +38,8 @@ static const std::map<uint8_t, sOperatingClass> operating_classes_list = {
     {119,       {{52, 60},                                                     beerocks::BANDWIDTH_40}},
     {120,       {{56, 64},                                                     beerocks::BANDWIDTH_40}},
     {121,       {{100, 104, 108, 112, 116, 120, 124, 128, 132, 136, 140, 144}, beerocks::BANDWIDTH_20}},
-    {122,       {{100, 108, 116, 124, 132, 140},                               beerocks::BANDWIDTH_40}},
-    {123,       {{104, 112, 120, 128, 136, 144},                               beerocks::BANDWIDTH_40}},
+    {122,       {{100, 108, 116, 124, 132, 136, 140},                          beerocks::BANDWIDTH_40}},
+    {123,       {{104, 112, 120, 128, 134, 136, 138, 144},                     beerocks::BANDWIDTH_40}},
```

Comment:
This fix should be in a separate commit.

Reply:
If I put the fix in a separate commit, then that commit will not pass existing unit tests. 
I think it's better to fix both the code and the unit tests in the same commit because they are tightly coupled. As a bonus, you can revert changes more easily if required. 

I can move the fix to a separate commit if there's a better reason to do it that I cannot think of. This is something to be decided and consistently applied because if we start writing unit tests, then it's going to happen very often.

Reply:
Ah, I forgot that the current test is incorrect as well...